pyproject.toml

```diff
@@ -18,6 +18,9 @@ langsmith = ">= 0.1.3, <= 0.1.49"
 
 [tool.poetry.group.dev.dependencies]
 openai = "*"
+faiss-cpu = "*"
+langchain = "*"
+langchain-openai = "*"
```

Comment:
Shouldn't we target a specific version here? I can imagine in the future interfaces will change, which we can avoid if we pin to specific versions.

Reply:
yea this is a good point, let me grab current versions

Reply:
fixed!